shared/router-v2/shim.native.js

```diff
@@ -11,18 +11,23 @@ const shimNewRoute = (Original: any) => {
   class ShimmedNew extends React.PureComponent<any, void> {
     static navigationOptions = Original.navigationOptions
     render() {
+      const navigationOptions =
+        typeof Original.navigationOptions === 'function'
+          ? Original.navigationOptions(this.props.navigation)
```

Comment:
The params should be different. Should be an object.  See the nav docs

Reply:
Oh! that's why I had issue with navigation being undefined. Thanks!